subprojects/workers/src/main/java/org/gradle/workers/IsolationMode.java

```diff
@@ -19,22 +19,26 @@
 import org.gradle.api.Incubating;
 
 /**
- * Forking mode for workers.
+ * Isolation mode for workers.
  *
  * @since 3.5
  */
 @Incubating
-public enum ForkMode {
+public enum IsolationMode {
     /**
      * Let Gradle decide, this is the default.
      */
     AUTO,
     /**
-     * Never fork, aka. use in-process workers.
+     * Don't attempt to isolate the work, use in-process workers.
      */
-    NEVER,
+    NONE,
```

Comment:
I'm not opposed at all to changing these, but I'm not sure I like these better.  I suppose "SHARED" would be ok.